src/lsdb/dask/merge_catalog_functions.py

```diff
@@ -170,17 +182,19 @@ def construct_catalog_args(
 
 def get_healpix_pixels_from_alignment(
     alignment: PixelAlignment,
-) -> Tuple[List[HealpixPixel], List[HealpixPixel]]:
-    """Gets the list of primary and join pixels as the HealpixPixel class from a PixelAlignment
+) -> Tuple[List[HealpixPixel], List[HealpixPixel], List[HealpixPixel]]:
+    """Gets the list of primary, join and aligned pixels as the HealpixPixel class from a PixelAlignment
 
     Args:
         alignment (PixelAlignment): the PixelAlignment to get pixels from
 
     Returns:
-        a tuple of (primary_pixels, join_pixels) with lists of HealpixPixel objects
+        a tuple of (primary_pixels, join_pixels, aligned_pixels) with lists of HealpixPixel objects
     """
     pixel_mapping = alignment.pixel_mapping
-    make_pixel = np.vectorize(HealpixPixel)
+    make_pixel = np.vectorize(
+        lambda o, p: HealpixPixel(o, p) if o is not None and p is not None else INVALID_PIXEL
```

Comment:
Change`align_catalog_to_partitions` to check for none and change this to none not invalid pixel